smdb/smdb/templates/base.html

```diff
@@ -7,115 +9,170 @@
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="{% url 'admin:index' %}">Admin</a>
-              </li>
-              {% if request.user.is_authenticated %}
-                <li class="nav-item">
-                  {# URL provided by django-allauth/account/urls.py #}
-                  <a class="nav-link" href="{% url 'users:detail' request.user.username  %}">{% trans "My Profile" %}</a>
-                </li>
-                <li class="nav-item">
-                  {# URL provided by django-allauth/account/urls.py #}
-                  <a class="nav-link" href="{% url 'account_logout' %}">{% trans "Sign Out" %}</a>
-                </li>
-              {% else %}
-                <li class="nav-item">
-                  {# URL provided by django-allauth/account/urls.py #}
-                  <a id="sign-up-link" class="nav-link" href="{% url 'account_signup' %}">{% trans "Sign Up" %}</a>
-                </li>
-                <li class="nav-item">
-                  {# URL provided by django-allauth/account/urls.py #}
-                  <a id="log-in-link" class="nav-link" href="{% url 'account_login' %}">{% trans "Sign In" %}</a>
-                </li>
-              {% endif %}
-            </ul>
-          </div>
-        </nav>
-      </div>
-      <div class="col-3">
-        <nav class="navbar-light float-right">
-          <a href="{{ MEDIA_URL }}logs/load.txt" target="_blank">load log output</a>
-        </nav>
-      </div>
+    <!-- NavBar -->
+    <!-- Left Menu Items -->
+    <div class="navbar navbar-expand-md navbar-light navbar-fixed-top navbar-left" style="background-color: #e9e9e9; width: 100%" id="navbar" role="navigation">
+        <a class="navbar-brand" href="{% url 'home' %}" id="smdb_navbrand">
+         <img  src="{% static 'images/SMDB_logo.jpg' %}" class="img" id="smdbLogo"
+          aria-label="SeaFloor Mapping Database (SMDB)">
+        </a>
+          <ul class="navbar-nav nav-pills w-100 justify-content-left flex-column flex-md-row" style="padding-left: 50px" width="50%" id="menuLinks">
+            <li class="nav-item active">
+              <a class="nav-link" id="navListActive" href="{% url 'home' %}" data-bs-toggle="tooltip" data-bs-placement="top" title="SMDB Home Page">
+                <font size="4px">Home</font><span class="sr-only"></span>
+              </a>
+            </li>
+
+            <li class="nav-item mr-auto dropdown">
+              <a class="nav-link dropdown-toggle" id="navData" href="" role="button" aria-haspopup="true" aria-expanded="false" data-bs-toggle="tooltip" data-bs-placement="top" title="View Database Sources">
+                <font size="4px">Data</font>
+              </a>
+                <ul class="dropdown-menu m-0 dropdown-content" aria-labelledby="navbarDropdown">
+                  <li><a class="dropdown-item" href="https://smdb.shore.mbari.org/missions/">Missions</a></li>
```

Comment:
Instead of hard-coded URLs use:
```
href="{% url 'missions' %}"
href="{% url 'expeditions' %}"
href="{% url 'compilations' %}"
```

Reply:
It looks like we actually need those names specified in urls.py to avoid a reverse lookup error. Here's the changes I needed to make:

```diff
➜  SeafloorMappingDB git:(esri) git diff smdb/config/urls.py
diff --git a/smdb/config/urls.py b/smdb/config/urls.py
index b0e6bdb..7f25c73 100644
--- a/smdb/config/urls.py
+++ b/smdb/config/urls.py
@@ -32,19 +32,19 @@ urlpatterns = [
     path("users/", include("smdb.users.urls", namespace="users")),
     path("accounts/", include("allauth.urls")),
     # Your stuff: custom urls includes go here
-    path("missions/", MissionTableView.as_view()),
+    path("missions/", MissionTableView.as_view(), name="missions"),
     path(
         "missions/<str:slug>/",
         MissionDetailView.as_view(),
         name="mission-detail",
     ),
-    path("expeditions/", ExpeditionTableView.as_view()),
+    path("expeditions/", ExpeditionTableView.as_view(), name="expeditions"),
     path(
         "expeditions/<str:slug>/",
         ExpeditionDetailView.as_view(),
         name="expedition-detail",
     ),
-    path("compilations/", CompilationTableView.as_view()),
+    path("compilations/", CompilationTableView.as_view(), name="compilations"),
     path(
         "compilations/<str:slug>/",
         CompilationDetailView.as_view(),
```